odoo/tools/misc.py

```diff
@@ -363,6 +363,27 @@ def add_sheet(self, name, cell_overwrite_ok=False):
     xlwt = None
 
 
+try:
+    import xlsxwriter
+
+    # add some sanitizations to respect the excel sheet name restrictions
+    # as the sheet name is often translatable, can not control the input
+    class PatchedXlsxWorkbook(xlsxwriter.Workbook):
+
+        def add_worksheet(self, name=None):
```

Comment:
Just take `**kwargs` and forward those to the underlying method? That way if the parent project adds parameters or whatever they will be properly propagated (at least for the keyword ones)